relayer/src/chain/cosmos.rs

```diff
@@ -181,7 +183,7 @@ impl CosmosSdkChain {
         // Gas Fee
         let coin = Coin {
             denom: "stake".to_string(),
-            amount: "1000".to_string(),
+            amount: "10000".to_string(),
```

Comment:
Is this supposed to stay hardcoded forever or do we eventually want to make this configurable/dynamic?

Reply:
@romac I believe this should be dynamic, we can have a default value but should allow the value to be passed as a parameter, I had opened a separate issue for that https://github.com/informalsystems/ibc-rs/issues/754